executor-scripts/linux/dhcp

```diff
@@ -14,20 +14,29 @@ determine_implementation() {
 start() {
 	case "$1" in
 	dhcpcd)
-		[ -n "$IF_HOSTNAME" ] && optargs="$optargs -h $IF_HOSTNAME"
-		[ -n "$IF_VENDOR" ] && optargs="$optargs -i $IF_VENDOR"
-		[ -n "$IF_CLIENT" ] && optargs="$optargs -i $IF_CLIENT"
-		[ -n "$IF_LEASETIME" ] && optargs="$optargs -l $IF_LEASETIME"
+		[ -n "$IF_DHCP_HOSTNAME" ] && optargs="$optargs -h $IF_DHCP_HOSTNAME"
+		[ -n "$IF_DHCP_VENDOR" ] && optargs="$optargs -i $IF_DHCP_VENDOR"
+		[ -n "$IF_DHCP_CLIENT_ID" ] && optargs="$optargs -i $IF_DHCP_CLIENT_ID"
+		[ -n "$IF_DHCP_LEASETIME" ] && optargs="$optargs -l $IF_DHCP_LEASETIME"
+
+		# Specific config file given?
```

Comment:
As mentioned in IRC, this section should be placed under `dhclient)` part.

Reply:
Fixed